index.js

```diff
@@ -164,6 +187,7 @@ NYC.prototype.cleanup = function () {
 
 NYC.prototype._createOutputDirectory = function () {
   mkdirp.sync(this.tmpDirectory())
+  mkdirp.sync(this._cacheDirectory())
```

Comment:
yes, but we pass allow `cacheDirectory` and `tempDirectory` options in the constructor. The collision is avoided with `temp` vs `tmp`, not sure how to pull that off elegantly here. Really I think we should rework option handling so that the option ends up being the same name as the accessor method.